libs/sdk-backend-tiger/src/backend/workspace/measures/index.ts

```diff
@@ -149,4 +153,42 @@ export class TigerWorkspaceMeasures implements IWorkspaceMeasuresService {
             });
         });
     }
+
+    public getMeasureReferencingObjects = async (ref: ObjRef): Promise<IMeasureReferencing> => {
+        const id = await objRefToIdentifier(ref, this.authCall);
+        const filterReferencingObj = {
+            filter: `metrics.id==${id}`, // RSQL format of querying data
+        };
+
+        const insights = await this.authCall((client) =>
+            MetadataUtilities.getAllPagesOf(
+                client,
+                client.workspaceObjects.getAllEntitiesVisualizationObjects,
+                {
+                    workspaceId: this.workspace,
+                },
+                { query: filterReferencingObj as any }, // return only measures that have a link to the given id in their visualizationObjects
+            ).then(MetadataUtilities.mergeEntitiesResults),
+        );
+
+        const filterReferencingObjX = {
+            filter: `metric.id==${id}`, // RSQL format of querying data
+        };
+        const measures = await this.authCall((client) =>
```

Comment:
Could we parallelise these 2 requests (1 for the measures, 1 for the insight) by Promise.all() ?